cmd/scw-wasm/main.go

```diff
@@ -0,0 +1,81 @@
+//go:build wasm && js
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"syscall/js"
+
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	"github.com/scaleway/scaleway-cli/v2/internal/namespaces"
+)
+
+var commands *core.Commands
+
+func getCommands() *core.Commands {
+	if commands == nil {
+		commands = namespaces.GetCommands()
+	}
+	return commands
+}
+
+func runCommand(args []string, stdout io.Writer, stderr io.Writer) chan int {
+	ret := make(chan int, 1)
+	go func() {
+		exitCode, _, _ := core.Bootstrap(&core.BootstrapConfig{
+			Args:      args,
+			Commands:  getCommands(),
+			BuildInfo: &core.BuildInfo{},
+			Stdout:    stdout,
+			Stderr:    stderr,
+			Stdin:     nil,
+		})
+		ret <- exitCode
+	}()
+
+	return ret
+}
+
+func wasmRun(this js.Value, args []js.Value) (any, error) {
+	cliArgs := []string{"scw"}
+	stdout := bytes.NewBuffer(nil)
+	stderr := bytes.NewBuffer(nil)
+
+	for argIndex, arg := range args {
+		if arg.Type() != js.TypeString {
+			return nil, fmt.Errorf("invalid argument at index %d", argIndex)
+		}
+		cliArgs = append(cliArgs, arg.String())
+	}
+
+	exitCodeChan := runCommand(cliArgs, stdout, stderr)
+	exitCode := <-exitCodeChan
+	if exitCode != 0 {
+		errBody := stderr.String()
```

Comment:
`stdout` might also be wanted here

Reply:
This error happen when a cli command is executed, it should not have something on stdout when an error occurs.
If a command return `value, err` and error is not nil then the error will be printed to stderr and the value will be dismissed.
What should be improved is using stderr when a command is successful, this is where warnings will be displayed.
Both output should be merged